habitat_baselines/config/default.py

```diff
@@ -101,17 +101,17 @@
 _C.RL.PPO.gamma = 0.99
 _C.RL.PPO.tau = 0.95
 _C.RL.PPO.reward_window_size = 50
-_C.RL.PPO.use_normalized_advantage = True
+_C.RL.PPO.use_normalized_advantage = False
 _C.RL.PPO.hidden_size = 512
 # -----------------------------------------------------------------------------
 # DECENTRALIZED DISTRIBUTED PROXIMAL POLICY OPTIMIZATION (DD-PPO)
 # -----------------------------------------------------------------------------
 _C.RL.DDPPO = CN()
 _C.RL.DDPPO.sync_frac = 0.6
 _C.RL.DDPPO.distrib_backend = "GLOO"
-_C.RL.DDPPO.rnn_type = "LSTM"
-_C.RL.DDPPO.num_recurrent_layers = 2
-_C.RL.DDPPO.backbone = "resnet50"
```

Comment:
@erikwijmans it looks like opposite to what is stated in PR description we are downgrading from resnet50 to resnet18 and from LSTM to GRU as default settings. Right?

Reply:
The DD-PPO configs still have it set to that, this is changing the defaults for things like `ppo_pointnav.yaml`